src/Microsoft.OData.Edm/Schema/EdmContainedEntitySet.cs

```diff
@@ -179,7 +179,7 @@ private IEdmPathExpression ComputePath()
                 newPath.Add(targetType.FullTypeName());
             }
 
-            newPath.Concat(this.NavigationPath.PathSegments);
+            newPath.AddRange(this.NavigationPath.PathSegments);

```

Comment:
>AddRange [](start = 20, length = 8)

I remember this change has been out in 7.4.3, Don't we?

Reply:
Yes; the PR ended up pulling in that commit, which has already been merged, but since it's the same commit it doesn't result in a conflict.